client/src/components/Login/Login.js

```diff
@@ -86,9 +89,21 @@ class Login extends Component {
                       <label htmlFor="password">Password</label>
                     </div>
                     <div className="form-group text-center">
-                      <button type="submit" className="btn btn-primary btn-lg">
-                        Login
-                      </button>
+                      {this.state.isDisabledButton ? (
+                        <button
+                          type="submit"
+                          className="btn btn-primary btn-lg"
+                        >
+                          Login
+                        </button>
+                      ) : (
+                        <button
```

Comment:
use react-bootstrap button component instead of the html that will save space and avoid duplication